packages/webapp/src/containers/CustomSignUp/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useDispatch } from 'react-redux';
+import PureCustomSignUp from '../../components/CustomSignUp';
+import { customSignUp } from './saga';
+
+function CustomSignUp() {
+  const { register, handleSubmit, errors, watch } = useForm({ mode: 'onBlur' });
```

Comment:
Can you tell me what `onBlur` is for?

Reply:
So that the in-line error message appears when a user clicks away from the input field.